### DISCUSSION TITLE
buddhism/8winds/

### DISCUSSION THREAD

**Discussion by giscus** [shane0/buddhism#31] [General]
# buddhism/8winds/

https://shanenull.com/buddhism/8winds/

**Comment by shane0** [comment]
If we have a hundred thousand, we want
a million. If we have a million, we want
ten million. Having ten million, we want a
billion; having a billion, we want the entire
country; having the entire country, we want
to conquer the world; having conquered the
world, we set our sights on outer space...
Human desire is endless and insatiable. By
attempting to satisfy it, our lives will be
immersed in greed, an ocean of suffering
from beginning to end.

**Comment by shane0** [comment]
https://youtu.be/9DCVJi4zTFc?si=pN7ZHQYm-qJu2l-E

**Comment by shane0** [comment]
https://youtu.be/S_1BEHtIpCM?si=wNyLGD6oy0Bc6KlB

**Comment by shane0** [comment]
Samyutta Nikaya 3.16: This sutta describes the eight worldly winds, which are prosperity, decline, disgrace, honor, praise, censure, suffering, and pleasure. The Buddha teaches that the eight worldly winds are like leaks that can cause us to suffer if we are not careful. When we are mindful of the eight worldly winds and develop wisdom and compassion, we can avoid being blown away by them and we can be free from suffering.
